### DISCUSSION TITLE
Migration Toolkit

### DISCUSSION THREAD

**Discussion by prasanna2400** [egovernments/Digit-Core#458] [DIGIT Platform]
Hi,

We are migrating MDMS v1 to MDMS v2. In the Postman collection, there is a folder for migration (endpoint /mdms-migration-toolkit/schema/v1/_migrate), but we are unable to find any service or script related to this in the project Git repository. Could you please share the repository or documentation link? That would be very helpful.

Thanks!

**Comment by Rishabh-egov** [comment]
Hi @prasanna2400
please find the relavant documentation [here](https://core.digit.org/platform/core-services/mdms-v2-master-data-management-service/mdms-migration) , it points to the service as well as the steps to use it for migrating from v1 to v2
